spec/features/user_adds_voter_to_list_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'user adds a voter to a list', %(
+As a signed up user
+I want to add a voter to a list
+So that I can manually add people who weren't captured by the initial search
+) do
+  # Acceptance Criteria
+  # [ ] A user can select a list from a voter's page and add that voter to
+  #     that list
+  # [ ] The user then receives a confirmation message and is taken to the
+  #     list page
+
+  scenario 'adds voter to list' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.